### DISCUSSION TITLE
Predicting When an Author Will Commit

### DISCUSSION THREAD

**Discussion by falahat** [datactive/bigbang#517] [Ideas]
Let's analyze for what sort of activity is associated with a committer who is about to add code in the next week or so. Some examples:
1. They might start emailing the mailing list a lot
2. Emails with different sentiment
3. Emails with certain phrases (ex. "I'm going to commit tomorrow.")

**Comment by sbenthall** [comment]
This is related to other work on participant tenure in IETF working groups.

We're doing a lot of work on the correlates/causaes of participation.

It's an interesting question if we can turn those estimates around and do prediction.

To test those predictions properly, we would need to implement backtesting -- i.e. use the historical data in a way that the prediction at time _t_ uses only data from prior to _t_.
